common/packages/parson/parson.cpp

```diff
@@ -811,7 +811,9 @@ static int json_serialize_string(const char *string, char *buf) {
     switch (c) {
       case '\"': APPEND_STRING("\\\""); break;
       case '\\': APPEND_STRING("\\\\"); break;
+#if PARSON_ENCODE_SLASH
       case '/':  APPEND_STRING("\\/"); break; /* to make json embeddable in xml\/html */
+#endif
```

Comment:
It doesn't look like this requires us to change/simplify any of our test cases?

Reply:
There should be no impact on our current tests.